src/blueapi/service/interface.py

```diff
@@ -49,12 +49,14 @@ def worker() -> TaskWorker:
 
 
 @lru_cache
-def messaging_template() -> StompClient | None:
-    stomp_config = config().stomp
+def cached_stomp_client() -> StompClient | None:
```

Comment:
Should: Why rename? And why rename only this function?

This seems like an unrelated change to the issue and should be reverted.

Reply:
it must have been left out when the main StompClient got changed

Reply:
PR here

and removed this change from this pyright PR
https://github.com/DiamondLightSource/blueapi/pull/673